llvm/lib/Target/EraVM/EraVMExpandSelect.cpp

```diff
@@ -134,13 +135,10 @@ bool EraVMExpandSelect::runOnMachineFunction(MachineFunction &MF) {
 
       if (ShouldInverse) {
         buildMOV(EraVM::ArgumentKind::In0, EraVMCC::COND_NONE);
```

Comment:
Please add here assert that we have an InverseCond for CCVal.

Reply:
Added assert.